app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users, class_name: 'User'
+  belongs_to :posts, class_name: 'Post'
+
+  def update_comments_counter
+    post.increment(:comments_counter)
+  end
+end
```

Comment:
When creating a comment, I get the following error: 
![image](https://github.com/ShiroYaksha90/blog-app/assets/93520178/08e60f25-655b-4df4-bfbb-7887ebc6e081)

This is probably because of the way you named the post when making the association. 
I suggest you do the following:
- On line 2: replace `:users` with `:author` 
- On line 3: replace `:posts` with `:post` and remove `class_name: 'Post'` as it is unnecessary.

[OPTIONAL] It would be great if you used `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to database, then make the `update_comments_counter` method private.